docs/troubleshoot/cli/known-cli.md

```diff
@@ -116,28 +129,50 @@ issue the following command:
 npm install npm@5.3.0 -g
 ```
 
-## Node.js commands do not respond as expected
+## Paths converting in Git Bash
 
-**Valid on Windows or Linux**
+**Valid on Windows**
 
 **Symptom:**
 
-You attempt to issue node.js commands and you do not receive the expected  output.
+When issuing a command with a relative directory path using forward slashes, Git Bash converts the path into an absolute path with a root directory.
```

Comment:
The path that gets converted is an absolute path. We could say something like this:
> When issuing a command with an absolute directory path that begins with a forward slash, Git Bash converts the path into an invalid one that does not exist.